server/service/src/sync/test/integration/central/mod.rs

```diff
@@ -56,24 +42,22 @@ async fn test_central_sync_record(identifier: &str, tester: &dyn SyncRecordTeste
             .await
             .expect("Problem deleting central data");
 
-        synchroniser.sync().await.unwrap();
-        check_integrated(&connection, step_data.integration_records)
+        site_config.synchroniser.sync(None).await.unwrap();
+        check_integrated(
+            &site_config.context.connection,
+            &step_data.integration_records,
```

Comment:
integration records passed as reference, made it easier in vaccination card test where it needs to be integrated and then checked in a couple of places